internal/pkg/model/changes.go

```diff
@@ -0,0 +1,139 @@
+package model
+
+import "sync"
+
+// LocalChanges contains all processed objects in local.UnitOfWork.
+type LocalChanges struct {
```

Comment:
2: Vytvoril som nove struktury `LocalChanges` a `RemoteChanges`, ktore sa budu pouzivat v `local/remote.UnitOfWork`.

Je to zoznam vsetkych operacii, ktore UOW spravilo, .... po dokonceni UOW sa to pouzije v `mapper`.

Povodne sa riesili iba `loaded`, `renamed`, ... teraz su pokryte vsetky operacie.
Je tak mozne implementovat aj scheduler `actiavate/deactivate` cez `mapper`.